pkg/config/types/publishers.go

```diff
@@ -37,6 +38,19 @@ type S3Publisher struct {
 	PreSignedURLExpiration Duration `yaml:"PreSignedURLExpiration,omitempty" json:"PreSignedURLExpiration,omitempty"`
 }
 
+type S3ManagedPublisher struct {
+	// BucketName specifies the S3 bucket name for managed storage
+	BucketName string `yaml:"BucketName,omitempty" json:"BucketName,omitempty"`
+	// KeyPrefix specifies an optional prefix for keys stored in the bucket
+	KeyPrefix string `yaml:"KeyPrefix,omitempty" json:"KeyPrefix,omitempty"`
```

Comment:
lets follow similar names to our current s3 publisher. Just use `Bucket` and `Prefix` instead of `BucketName` and `KeyPrefix`